compose/multiple-compose-files/include.md

```diff
@@ -0,0 +1,37 @@
+---
+description: How to use Docker Compose's include top-level element
+keywords: compose, docker, include, compose file
+title: Include
+---
+
+With Docker Compose version 2.20 and later, you can include a whole Compose file directly in your local Compose file using the [`include` top-level element](../compose-file/14-include.md). This solves the relative path problem that [`extends`](extends.md) and [merge](merge.md) present. 
+
+`include` makes it easier to modularize complex applications into sub-Compose files. This allows application configurations to be made simpler and more explicit. This also helps to reflect in the config file organization the engineering team responsible for the code.
+
+Each path listed in the `include` section loads as an individual Compose application model, with it’s own project directory, in order to resolve relative paths.
+
+Once the included Compose application loads, all resources definitions are copied into the current Compose application model.
+
+> **Note**
+>
+> `include` applies recursively so an included Compose file which declares its own `include` section, triggers those other files to be included as well.
```

Comment:
Wondering if we can simplify this note?  "..triggers those other files" Sounds slightly confusing.